src/domain/payments/payment-flow-builder.ts

```diff
@@ -491,6 +500,13 @@ const LPFBWithConversion = <S extends WalletCurrency, R extends WalletCurrency>(
     return state.usdPaymentAmount
   }
 
+  const skipProbeForDestination = async () => {
+    const state = await statePromise
```

Comment:
why the other methods use `const state = await Promise.resolve(statePromise)` ?

Reply:
Oh I thought I already standardized this. I believe the `Promise.resolve(statePromise)` is redundant. We can probably change all instances of this to be simpler.

Reply:
yup I noticed this too lets make it consistent

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1505/commits/11f8119a64597b1fc695e346e5c44b10502d7b9f